packages/fxa-graphql-api/package.json

```diff
@@ -34,6 +34,7 @@
     "apollo-server": "^2.13.1",
     "apollo-server-express": "^2.13.1",
     "convict": "^6.0.0",
+    "fxa-js-client": "^1.0.25",
```

Comment:
👀 should this the new client or at least reference the client in the local repo? I Think the npm version is gonna be deprecated

Reply:
Oh, I don't know. @dannycoates?

Reply:
There's a change that' s upcoming that makes some large changes to how things reference other bits in the local repo. So it'd probably be better to wait for that to land then switch to it.

Reply:
yeah, we should use "workspace:*" for this after yarn lands

Reply:
in the meantime, this is fine

Reply:
My next PR will need to immediately switch to the version in this monorepo as the NPM release does not have `attachedClients` which is needed.